playground/hmr/prune/dep1.js

```diff
@@ -0,0 +1,7 @@
+import './dep2.js'
+
+// TODO: currently we need one more `import` in this module
```

Comment:
Let's add a link to the issue you are going to create here.

Reply:
Updated with https://github.com/vitejs/vite/issues/20781